### PR TITLE
Blog Post: Dynamic Ingress in Kubernetes

### PR DESCRIPTION
**Pull request by kbarnard10**
Blog and images for post.

### CODE REVIEW

**Comment by kbarnard10** [comment]
Can one of you lgtm and approve this? 

/assign @SarahKConway 
/assign @natekartchner 
/assign @zacharysarah 
/hold

File: content/en/blog/_posts/2018-06-01-dynamic-ingress-kubernetes.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+---
+layout: blog
+title: Dynamic Ingress in Kubernetes
+date:  Friday, June 1, 2018
+---
+
+**Author**: Richard Li (Datwaire)
```
Comment (sublimino) [inline content/en/blog/_posts/2018-06-01-dynamic-ingress-kubernetes.md]: should this be "Richard Li (Datawire)"?

**Pull request merged**